community/modules/compute/schedmd-slurm-gcp-v5-node-group/README.md

```diff
@@ -36,50 +36,6 @@ module as input with:
     partition_name: compute
 ```
 
-### Compute VM Zone Policies
```

Comment:
I'm a little concerned that this documentation may be disappearing. Unclear if accidental or intentional due to #662

Reply:
The same content exists already in community/modules/compute/schedmd-slurm-gcp-v5-partition/README.md, which is where zone policies are defined. 
It was mistakenly added in both places in a prior PR, and in retrospect it should have been mentioned in the description.